config/environments/development.rb

```diff
@@ -77,4 +77,7 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+  config.hosts << ".ngrok-free.app"
```

Comment:
How about using an environment variable for this so it doesn't need to be changed across developer environments:

```suggestion
  config.hosts << ENV["DEV_HOSTS"]
```

Reply:
@littleforest where do we configure the environment variables? I see we are using the dotenv-rails gem but I can't find any .env file that stores env.

Reply:
can I instead store it in the credentials/development.yml.enc file?

Reply:
@xihai01 you can just create a `.env` file in the root of the app and add the `DEV_HOSTS` environment variable. Just be sure to restart the server once you create and add things to this file.